src/moscot/utils/subset_policy.py

```diff
@@ -377,18 +512,20 @@ def create_policy(
 
     Parameters
     ----------
-    %(policy_kind)s
-    %(adata)s
+    kind
+        What policy to create.
+    adata
+        Annotated data object.
     kwargs
-        Keyword arguments for a :class:`~moscot.utils.subset_policy.SubsetPolicy`.
+        Additional keyword arguments.
 
     Returns
     -------
     The policy.
 
     Notes
     -----
-    TODO: link policy example.
+    TODO(michalk8): link policy example.
```

Comment:
@ArinaDanilina new issue for this please

Reply:
#561